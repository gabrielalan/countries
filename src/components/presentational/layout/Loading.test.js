import React from 'react';
import renderer from 'react-test-renderer';
import Loading from './Loading';

describe('Loading', () => {
  it('should render the required html structure', () => {
    const tree = renderer.create(<Loading id="clear" />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
