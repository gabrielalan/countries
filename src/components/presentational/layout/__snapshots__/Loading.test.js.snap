// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loading should render the required html structure 1`] = `
<div
  className="loading"
>
  <div
    className="loading__icon"
  >
    <div />
    <div />
    <div />
    <div />
    <div />
    <div />
    <div />
    <div />
  </div>
</div>
`;
